### ISSUE TITLE
FPC build fails for trunk, stable, fixes from gitlab for Windows 64

### ISSUE THREAD

**Issue by dmz73vu** [LongDirtyAnimAlf/fpcupdeluxe#407]
There seems to be a problem building for Windows 64 bit.
Error message:
version.pas(94,8) Error: Operator is not overloaded: "ShortString" + "Word"
version.pas(100) Fatal: There were 1 errors compiling module, stopping
Fatal: Compilation aborted

from version.pas:
function full_version_string:string;
begin
  full_version_string := version_nr+'.'+release_nr+'.'+patch_nr+minorpatch
{$ifdef REVINC}
  +'-r'+{$i revision.inc} // <==== ERROR LINE
{$endif REVINC}
  ;
end;

revision.inc: 
45835

Notice there are not single quotes around the revision number.

I can see message in the log that says:
echo '45835' > revision.inc

Not sure if the issue is with fpcupdeluxe or with FPC.

**Comment by LongDirtyAnimAlf** [comment]
This sometimes happens when you have a mingw, msys or cygwin "echo" in the path.
Fpcupdeluxe tries to isolate from these installs, but this sometimes fails.
So, its neither an issue of FPC or fpcupdeluxe.
Please report back if this info solves.

**Comment by dmz73vu** [comment]
Thanks for the response. 
I am aware of the issues with other build tools on the path and I have been running the fpcupdeluxe with "clean" path that only includes the following: C:\WINDOWS\system32;SystemRoot\System32\Wbem;SystemRoot\System32\WindowsPowerShell\v1.0
The reason I suspected either FPC or fpcupdeluxe is because the previous version of fpcupdeluxe (I think 2.0.0e) has worked with this clean path but then I updated to fpcupdeluxe 2.0.1 and latest FPC source and I started getting these errors.
Granted, I have also applied latest Windows updates so it could be that but I still think the most likely culprit is fpcupdeluxe since it had some significant changes since the last build that worked.
I am unable to find fpcupdeluxe 2.0.0e to try again with that version so I decided to report the issue here in case there are other people with similar problem.
